src/styles/blocks/header.scss

```diff
@@ -0,0 +1,86 @@
+body {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+
+  margin-left: 50px;
+}
+
+.logo__link {
+  display: inline-block;
+}
+
+.logo__moyo {
+  display: block;
+
+  height: 100%;
+  width: 100%;
+}
+
+.nav {
+  display: inline-block;
+
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+}
+
+.nav__link {
+  display: inline-block;
+
+  line-height: 60px;
+
+  text-decoration: none;
+  color: #060b35;
+  text-transform: uppercase;
+}
```

Comment:
move `&__list` to block `.nav`. 
Also for other block and element. 
You need use block selector ( in this case it `.nav`) and inside write all elements and modifiers ( in this case it `&__list`, `&__link` and other.
Please, check all files*

Reply:
fixed